force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -69,7 +72,10 @@ public with sharing class VariablesDataTypesOperators {
      * @return The quotient of the division, or 0 if the denominator is zero.
      */
     public static Decimal division(Integer a, Integer b) {
-        return null; // Replace null with the variable you used to store the result
+        if(b != 0){
+            return a/b;
```

Comment:
What happens when a is not perfectly divisible by b?

Reply:
Since a and b are both integers, it will return a/b as a whole number (rounded down). Right?